internal/clients/manager.go

```diff
@@ -173,6 +184,23 @@ func (c *AdminAPIClientsManager) GatewayClients() []*adminapi.Client {
 	return lo.Values(c.readyGatewayClients)
 }
 
+func (c *AdminAPIClientsManager) GatewayClientsToConfigure() []*adminapi.Client {
```

Comment:
This should have a godoc. It might be very hard to distinguish a subtle difference between this and `GatewayClients()`

Reply:
added docs.